scanner_id: "1711004"
frame_id: "PhoXi3Dscanner_sensor"
latch_tipics: true
topic_queue_size: 1
init_from_config: false # if true all following parameters will be inicialized from this config otherwise from PohXi control application.
# All following parameters are for PhoXi Control and they can override all dynamic_reconfigure parameters in cfg file.
# This values are set to scanner after successful connection only if init_from_config parameter is true.
coordination_space: 1 # 1 = Camera, 2 =  Mounting, 3 = Marker, 4 = Robot, 5 = Custom
resolution: 1         # 0 = Low, 1 = High
scan_multiplier: 1
confidence: 3.0
send_confidence_map: true
send_deapth_map: true
send_normal_map: true
send_point_cloud: true
send_texture: true
shutter_multiplier: 1
timeout: -3          # in ms, special parameters: 0 = Zero, -1 = Infinity, -2 = Last stored, -3 = Default
trigger_mode: 1      # 0 = Free run, 1 = Software
